connector/tests/src/test/java/oaebudt/dataspace/connector/utils/OaebudtParticipant.java

```diff
@@ -0,0 +1,132 @@
+
+    private OaebudtParticipant() {
+
+    }
+
+    public ServiceExtension seedVaultKeys() {
+        return new SeedVaultKeys();
+    }
+
+    public static class Builder extends Participant.Builder<OaebudtParticipant, Builder> {
+
+        public static Builder newInstance() {
+            return new Builder(new OaebudtParticipant());
+        }
+
+        protected Builder(final OaebudtParticipant participant) {
+            super(participant);
+        }
+    }
+
+    public Config getConfiguration(final int vaultPort, final int postgresPort) {
+        final var map = Map.ofEntries(
+                entry("edc.participant.id", id),
+                entry("web.http.path", "/api"),
+                entry("web.http.port", getFreePort() + ""),
+                entry("web.http.control.path", "/control"),
+                entry("web.http.control.port", getFreePort() + ""),
+                entry("web.http.management.path", controlPlaneManagement.get().getPath()),
+                entry("web.http.management.port", controlPlaneManagement.get().getPort() + ""),
+                entry("web.http.protocol.path", controlPlaneProtocol.get().getPath()),
+                entry("web.http.protocol.port", controlPlaneProtocol.get().getPort() + ""),
+                entry("web.http.version.path", "/version"),
+                entry("web.http.version.port", getFreePort() + ""),
+                entry("web.http.public.path", "/public"),
+                entry("web.http.public.port", getFreePort() + ""),
+                entry("edc.transfer.proxy.token.verifier.publickey.alias", "public-key-alias"),
+                entry("edc.transfer.proxy.token.signer.privatekey.alias", "private-key-alias"),
+                entry("edc.iam.issuer.id", "http://mock-issuer"),
+                entry("edc.api.auth.key", "password"),
+                entry("edc.iam.sts.publickey.id", "dummy-key"),
+                entry("edc.iam.sts.publickey.alias", "dummy-key"),
+                entry("edc.iam.sts.privatekey.alias", "dummy-key"),
+                entry("web.http.catalog.port", getFreePort() + ""),
+                entry("web.http.catalog.path", "/catalog"),
+                entry("edc.runtime.id", id),
+                entry("edc.dpf.selector.url", "http://localhost:" + getFreePort() + "/control/v1/dataplanes"),
+                entry("edc.vault.hashicorp.url", "http://localhost:" + vaultPort),
+                entry("edc.vault.hashicorp.token", "root"),
+                entry("edc.vault.hashicorp.health.check.enabled", "false"),
+                entry("edc.datasource.default.url", "jdbc:postgresql://localhost:" + postgresPort + "/db"),
+                entry("edc.datasource.default.user", "password"),
+                entry("edc.datasource.default.password", "password"),
+                entry("edc.sql.schema.autocreate", "true")
+        );
+
+        return ConfigFactory.fromMap(map);
+    }
+
+    @Override
+    public RequestSpecification baseManagementRequest() {
```

Comment:
no need to override

Reply:
I had to override so I can put the api key in the request header